docs/package.json

```diff
@@ -25,16 +25,17 @@
     "@emotion/server": "11.11.0",
     "@emotion/styled": "11.14.0",
     "@mui/base": "5.0.0-beta.68",
-    "@mui/docs": "6.3.1",
-    "@mui/icons-material": "6.3.1",
+    "@mui/docs": "6.4.1",
+    "@mui/icons-material": "6.4.1",
     "@mui/internal-markdown": "1.0.24",
     "@mui/joy": "5.0.0-beta.51",
     "@mui/lab": "6.0.0-beta.22",
-    "@mui/material": "6.3.1",
+    "@mui/material": "6.4.1",
     "@mui/material-nextjs": "6.3.1",
-    "@mui/styles": "6.3.1",
-    "@mui/system": "6.3.1",
-    "@mui/utils": "6.3.1",
+    "@mui/styles": "6.4.1",
+    "@mui/system": "6.4.1",
+    "@mui/utils": "6.4.1",
+    "@mui/x-data-grid-pro": "7.26.0",
```

Comment:
Adding Data Grid pro for one of the examples in the docs. 
Should we skip this? A license might be needed not to show the watermark too.

Reply:
I think it's a good example to demonstrate how you could combine the Crud component with paid X Datagrid offerings, I'd vote to keep that example there